tests/unit/test_graph.py

```diff
@@ -73,12 +73,13 @@ def test_remove_switch_hops(self):
                                      "00:00:00:00:00:00:00:01:2"]}
         self.assertEqual(circuit, expected_circuit)
 
-    @patch('networkx.shortest_simple_paths', return_value=["any"])
-    def test_shortest_paths(self, mock_shortest_simple_paths):
-        """Test shortest paths."""
-        source, dest = "00:00:00:00:00:00:00:01:1", "00:00:00:00:00:00:00:02:2"
-        shortest_paths = self.kytos_graph.shortest_paths(source, dest)
+    # @patch('networkx.shortest_simple_paths', return_value=["any"])
+    # def test_shortest_paths(self, mock_shortest_simple_paths):
+    #    """Test shortest paths."""
+    #    source, dest = "00:00:00:00:00:00:00:01:1",
+    # "00:00:00:00:00:00:00:02:2"
+    #    shortest_paths = self.kytos_graph.shortest_paths(source, dest)
 
-        mock_shortest_simple_paths.assert_called_with(self.kytos_graph.graph,
-                                                      source, dest, None)
-        self.assertEqual(shortest_paths, ["any"])
+    #    mock_shortest_simple_paths.assert_called_with(self.kytos_graph.graph,
+    #                                                  source, dest, None)
+    #    self.assertEqual(shortest_paths, ["any"])
```

Comment:
We commented them out because they called `_set_default_metadata(...)`, which no longer exists in our implementation of graph.py.

They were not moved. 

I can add the method back, but I will need to research how to do that without impacting our revision history.

Reply:
Your revision history will probably be squashed into one bit commit before merging. If you want us to maintain it somehow, try to reduce it to just a handful of commits by `git rebase`'ing/squashing it yourself.

Reply:
I have readded the commented-out tests.